core/interfaces.py

```
@@ -0,0 +1,290 @@
+
+
+class IManifestManager(ABC):
+    """Manifest 管理器接口"""
+
+    @abstractmethod
+    def load_manifest(self) -> Optional[Manifest]:
+        """
+        加载 manifest
+
+        Returns:
+            Manifest 对象，如果不存在则返回 None
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)